generators/app/templates/.editorconfig

```diff
@@ -8,7 +8,7 @@ indent_size = 2
 insert_final_newline = true
 trim_trailing_whitespace = true
 end_of_line = lf
```

Comment:
This is what the ionic starters have, and I didn't want anything I borrowed from ionic to flag here